src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -11,12 +12,10 @@ function renderPaginationLink(data, index){
 	// CONTEXT
 	// --------------------------
 	const { 
-		get: {
+		state: {
 			activeIndex 
 		},
-		set: {
-			activeIndex: setActiveIndex
-		}
+		dispatch
 	} = useContext(Episodes);
```

Comment:
`.state`

Reply:
`.state` wouldn't help here as we're destructuring dispatch also, which is state's sibling.